content/post/03-keepassxc-freedesktop-secret.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Using KeePassXC as your system-keyring and ssh-agent"
+date: 2019-11-04T11:33:37-03:00
+draft: false
+---
+
+# Introduction
+
+> Guest article by maxice8
+
+Recently the community-oriented password manager [KeePassXC](https://keepassxc.org/) made a new
+major release, 2.5.0.
+
+In this new release there was a really important change, the implementation of the
+[secrets](https://freedesktop.org/wiki/Specifications/secret-storage-spec/secrets-api-0.1.html)
+from Freedesktop.
+
+> This is oversimplified
+
+For those that don't know, the secrets API is a specification for
+storing and querying secrets (duh) like passwords, tokens and ssh keys, and really
+anything you wanted.
+
+After getting pleasantly surprised that someone else updated the keepassxc package on
+Alpine Linux, and sorting a small error in where the keepassxc build system doesn't
+error out if freedesktop integration is enabled but the required libsecret-dev package
+isn't present, i decided to enable keepassxc as my system-keyring instead of gnome-keyring
+
+# Setting up within keepassxc
+
+First thing to do is enabling the `Secret Service Integration` on keepassxc itself, this can
+be done by clicking on `tools` in the bar on top, then `settings` and scrolling down the left bar
+and clicking on `Secret Service Integration`. Then tick
```

Comment:
Shouldn't tools ans settings be capatilised?

Reply:
yes